tsconfig.json

```diff
@@ -13,7 +13,6 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": true,
-    "strictNullChecks": true,
```

Comment:
I turned this off to get things compiling, but it would probably be good to turn it back on